webroot/js/utils/constants.js

```diff
@@ -27,7 +27,26 @@ export const KEY_CHAT_FIRST_MESSAGE_SENT = 'owncast_first_message_sent';
 export const CHAT_INITIAL_PLACEHOLDER_TEXT = 'Type here to chat, no account necessary.';
 export const CHAT_PLACEHOLDER_TEXT = 'Message';
 export const CHAT_PLACEHOLDER_OFFLINE = 'Chat is offline.';
-
+export const CHAT_MAX_MESSAGE_LENGTH = 500;
+export const CHAT_CHAR_COUNT_BUFFER = 20;
+export const CHAT_OK_KEYCODES = [
+  'ArrowLeft',
+  'ArrowUp',
+  'ArrowRight',
+  'ArrowDown',
+  'Shift',
+  'Meta',
+  'Alt',
+  'Delete',
+  'Backspace',
+];
+export const CHAT_KEY_MODIFIERS = [
+  'Control',
+  'Shift',
+  'Meta',
+  'Alt',
+];
+export const MESSAGE_JUMPTOBOTTOM_BUFFER = 260;
```

Comment:
I'm finding that using such a fixed number doesn't work if the next message is really long. Might need to try the other option outlined in #101  🤔 

On a related note, should we cap the number of lines people can type in?

Reply:
I have no opinion on if we should cap the number of lines.  I could see somebody being annoying by typing a single character per line for hundreds of lines, but it's probably rare.